### ISSUE TITLE
The client has a large CPU usage！

### ISSUE THREAD

**Issue by JCCD** [Gr1mmie/AtlasC2#2]
Hey man，when I use your C2 i Find the implant will use  a large CPU usage.
And I don't think it's a good Idea.Like this Picture.
![cpu](https://user-images.githubusercontent.com/39368758/159893999-27ceedbd-784b-44bd-8596-96a17719fb60.png)

**Comment by Gr1mmie** [comment]
Yeah, that's def no bueno. I did have one memory overflow issue in my testing but nothing about CPU usage, I'll have to look into this later. Is this just being run on idle?

**Comment by JCCD** [comment]
Yes, I tried to simulate how C2 would be placed in a real world and discovered this problem.

**Comment by Gr1mmie** [comment]
I just pushed a commit that should fix this issue, had to do with the implant loop that checked for tasks was running without any pauses. As soon as I gave it a 1 second sleep in between checks, the power usage dropped down to Very low and the CPU usage dropped down to 0% when it was 12 - 15% before. I also set it so that the implant process changes to `Idle` on startup.

![image](https://user-images.githubusercontent.com/57014148/160019904-41e3d427-4778-47b9-be67-0177e3c58417.png)
